### ISSUE TITLE
Gallery error message when dragging single video file into editor

### ISSUE THREAD

**Issue by roundhill** [WordPress/gutenberg#35366]
### Description

When uploading a video .mp4 file by dragging it into the editor, this error message appears:

<img width="1077" alt="Screen Shot 2021-10-05 at 11 34 42 AM" src="https://user-images.githubusercontent.com/789137/136081382-61e8a984-1ab4-4686-b1cd-3ac8ee58fa87.png">

### Step-by-step reproduction instructions

1. Find a video file on your machine to upload to a post.
2. Drag and drop it into the editor.
3. Observe the gallery error message, which is out of context for the current action. It probably shouldn't display for a single video file?

### Environment info

Safari, macOS 11.6.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by stevenlinx** [comment]
Is it just me?

Isn't this the expected behavior?

The gallery block is only for images.
https://gogutenberg.com/blocks/gallery/

**Comment by roundhill** [comment]
It doesn't make sense to me to see that error message, because when uploading a single video there is no intention to add it to a gallery.

**Comment by stevenlinx** [comment]
1.)
This point here is only reference for others reading.

Gallery block is only for images:
https://wordpress.com/support/wordpress-editor/blocks/gallery-block/
"The Gallery block allows you to easily add multiple photos and have them automatically arranged in an attractive way." (it doesn't mention video)

To embed videos, you can only use the Video block.
https://wordpress.com/support/wordpress-editor/blocks/video-block/
"The video block allows you to upload and embed video into your post or page."

2.)
@roundhill : "...when uploading a single video there is no intention to add it to a gallery."
You're saying you want to just drag a video file to "an empty space" of the editor and have the editor automatically embed as a Video block for you? (i.e. you don't want to add the video to the gallery block to begin with).

In other words, you're trying to say the bug happens when the editor automatically choose a Gallery block for the media dragged into instead of the expected Video block (if the media is a video)?

In your setup, does manually choosing a Video block work?

**Comment by talldan** [comment]
I tested and it worked ok for me, a video block was inserted when dragging to an empty part of the editor. @roundhill could this issue be caused by a plugin that you have active?

**Comment by glendaviesnz** [comment]
I was able to replicate this. The video block was added as expected but the error still displayed:
<img width="1170" alt="Screen Shot 2021-10-19 at 4 53 45 PM" src="https://user-images.githubusercontent.com/3629020/137841224-d779db47-b0ed-459f-aabd-af15d1d7c017.png">

I think it is related to some of the changes with the refactored gallery block so have added to that project board.

**Comment by glendaviesnz** [comment]
I have added a potential fix for this at https://github.com/WordPress/gutenberg/pull/35790
